src/app/pages/Itinerary/Itinerary.tsx

```diff
@@ -19,6 +20,21 @@ export default function Itinerary(): JSX.Element {
     );
   }
 
+  async function deleteDestination(id: ObjectId) {
+    const response = await fetch(`api/location/${id}`, {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    if (response.status === 200) {
+      console.log('Destination added to DB!');
+    } else {
+      console.log('An error occured =(');
```

Comment:
do you need the logs, or maybe its better to integrate smth for the user to read?